robottelo/config/base.py

```diff
@@ -638,15 +639,16 @@ def read(self, reader):
         self.image_username = reader.get('rhev', 'image_username')
         self.image_password = reader.get('rhev', 'image_password')
         self.image_name = reader.get('rhev', 'image_name')
+        self.ca_cert = reader.get('rhev', 'ca_cert')
```

Comment:
will this work even if i don't use a setup with SSL an thus i don't have any cert that i would provide?

Reply:
Done, Made the param optional